### PR TITLE
appveyor: build more things, also for x64 and publish the build results

### PR DESCRIPTION
**Pull request by lazka**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -7,14 +7,22 @@ environment:
   global:
     DOWNLOAD_DIR: _download_cache
     APPVEYOR_SAVE_CACHE_ON_ERROR: true
-    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd
+    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd;C:\Program Files\7-Zip
   matrix:
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```
Comment (nacho) [inline appveyor.yml]: This should be named 2015?

File: appveyor.yml [hunk 2]
```diff
@@ -7,14 +7,22 @@ environment:
   global:
     DOWNLOAD_DIR: _download_cache
     APPVEYOR_SAVE_CACHE_ON_ERROR: true
-    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd
+    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd;C:\Program Files\7-Zip
```
Comment (nacho) [inline appveyor.yml]: why do we need 7zip? Is that something that comes by default? If not we can use the tar that we have from msys2

File: appveyor.yml [hunk 3]
```diff
@@ -7,14 +7,22 @@ environment:
...
   matrix:
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
       VS_VER: "14"
       PLATFORM: "x86"
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```
Comment (nacho) [inline appveyor.yml]: same here 2015?

I also think we should probably build for 2015 and 2017 (can be in a different patch though)

File: appveyor.yml (hunk 2 above)
Comment (lazka) [reply appveyor.yml]: It's part of the default appveyor install and used below in build_script. Sure, will switch to msys2

File: appveyor.yml (hunk 1 above)
Comment (lazka) [reply appveyor.yml]: The "Visual Studio 2017" image has both VS2015 and VS2017 installed, see here for details: https://www.appveyor.com/docs/build-environment/#image-updates

The only upside from 2015 is afaics that it uses an older Windows and would fail on unintended Window 10 only API usage.

File: appveyor.yml (hunk 3 above)
Comment (lazka) [reply appveyor.yml]: see previous reply.

**Comment by lazka** [comment]
done

**Comment by nacho** [comment]
Thanks
